docs/zh_CN/Compressor/Pruner.md

```diff
@@ -102,16 +102,6 @@ pruner.compress()
 
 ### 用法
 
-TensorFlow 代码
-```python
-from nni.compression.tensorflow import FPGMPruner
-config_list = [{
-    'sparsity': 0.5,
-    'op_types': ['Conv2D']
-}]
-pruner = FPGMPruner(model, config_list)
-pruner.compress()
-```
```

Comment:
i think we should keep Chinese doc unchanged, and let @JunweiSUN to update Chinese doc using crowdin